app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2>All Users</h2>
+<section class="users-list">
+
+  <% @users.each do |user| %>
+    <section id="user-<%= user.id %>">
+      <%= link_to user.name, "/admin/users/#{user.id}" %> -
+      Created At: <%= user.created_at %> -
+      Role: <%= user.role %>
```

Comment:
Should we include more information here about the user such as his address?

Reply:
Not asked for in the user story. Each name is a link to their individual page though so they also see address and email from there.